src/utils/string/StringUtils.js

```diff
@@ -3,6 +3,7 @@
 import * as React from 'react';
 import moment from 'moment';
 import reactStringReplace from 'react-string-replace';
+import _, {isArray} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for lodash seems to be importing the entire library just to use `isArray`. This could potentially increase the bundle size of your application. Consider importing only the required function.
```diff
- import _, {isArray} from 'lodash';
+ import isArray from 'lodash/isArray';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->